berlin/css/pyladies.css

```diff
@@ -15,3 +21,55 @@ section.about_us, section.resources {
 section.about_us img {
    width: 500px; 
 }
+
+.logo, .nav {
+//  text-align: center;
+}
+
+.logo img {
+  margin: 0 auto;
+}
+
+h1 {
+//  text-align: center;
+  font-size: 115%;
+  text-transform: uppercase;
+}
+
+video {
+  width: 100%;
+  cursor: pointer;
+  background-image: url("img/vidprelo.jpg");
+}
+video:hover {
+  background-image: url("img/vidpre.jpg");
```

Comment:
I think this and the background-image url a few lines above are giving a 404 - it looks like it should actually be `url("/img/vidpre.jpg")` (with the preceding `/`).
![2015-05-28 at 2 51 pm](https://cloud.githubusercontent.com/assets/1191069/7868316/231922a8-0549-11e5-96d5-14eb13df2367.png)